geode-core/src/distributedTest/java/org/apache/geode/internal/cache/partitioned/PersistentPartitionedRegionDistributedTest.java

```diff
@@ -1606,6 +1611,23 @@ private void revokeKnownMissingMembers(final int numExpectedMissing)
     }
   }
 
+  private static InetAddress getFirstInet4Address() throws SocketException, UnknownHostException {
+    Enumeration<NetworkInterface> nets = NetworkInterface.getNetworkInterfaces();
+    InetAddress firstInet4Address = InetAddress.getLocalHost();
```

Comment:
Can we delete this `firstInet4Address` variable as it's not used within the entire method?.

Reply:
Yes this was the result of flipping back and forth between two implementations and missing a spot.  Thank you!